fhir-search/src/main/java/com/ibm/fhir/search/parameters/ParametersMap.java

```diff
@@ -97,7 +133,21 @@ public int size() {
         return Collections.unmodifiableSet(codeMap.entrySet());
     }
 
+    /**
+     * @deprecated use {@link #canonicalEntries()} instead
+     */
+    @Deprecated
     public Set<Entry<String, SearchParameter>> urlEntries() {
-        return Collections.unmodifiableSet(urlMap.entrySet());
+        return Collections.unmodifiableSet(canonicalMap.entrySet().stream()
+                .filter(e -> !e.getKey().contains("|"))
+                .collect(Collectors.toSet()));
+    }
```

Comment:
Maybe just remove these deprecated methods? No other team should be depending on this behavior.  While observable, it shouldn't be a problem.

Reply:
i was trying to preserve backwards compatibility in the java where possible, but i'm ok removing them now if we don't want to wait

Reply:
I  vote for removing

Reply:
done